net-wireless/unifi/unifi-7.1.61-r1.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Set this var for any releases except stable
+RC_SUFFIX="-c7eb1400e2"
+
+inherit java-pkg-2 readme.gentoo-r1 systemd
+
+DESCRIPTION="A Management Controller for Ubiquiti Networks UniFi APs"
+HOMEPAGE="https://www.ubnt.com"
+SRC_URI="https://dl.ui.com/unifi/${PV}${RC_SUFFIX}/UniFi.unix.zip -> ${P}.zip"
+S="${WORKDIR}/UniFi"
+
+KEYWORDS="-* ~amd64 ~arm64"
+LICENSE="Apache-1.0 Apache-2.0 BSD-1 BSD-2 BSD CDDL EPL-1.0 GPL-2 LGPL-2.1 LGPL-3 MIT ubiquiti"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="systemd system-mongodb"
+REQUIRED_USE="system-mongodb? ( systemd )"
```

Comment:
What about OpenRC? Couldn't be provide an updated init script with ```rc_need mongodb``` and ```rc_after mongodb```?

Reply:
Added OpenRC support